docs/dev_install.rst

```diff
@@ -1,20 +1,38 @@
 Installation for development
 ============================
 
-.. code-block:: bash
+#. Clone repo and install requirements:
 
-    $ sudo apt-get install virtualenvwrapper
-    $ mkvirtualenv mmpy_bot
-    $ git clone https://github.com/attzonko/mmpy_bot.git
-    $ cd mmpy_bot
-    $ python setup.py develop
-    $ pip install -r requirements.txt
-    $ pip install -r docs/requirements.txt
-    $ touch local_settings.py               # configure your local settings
-    $ mmpy_bot                              # run bot
+    .. code-block:: bash
 
+        $ git clone https://github.com/attzonko/mmpy_bot.git
+        $ cd mmpy_bot
+        $ pip install -r requirements.txt
+        $ pip install -r dev-requirements.txt
+
+#. Spin up the Mattermost container (Podman or Docker required):
+
+    .. code-block:: bash
+
+        $ podman-compose -f tests/integration_tests/docker-compose.yml up -d
+
+#. Edit mmpy_bot/settings.py and configure as below. If testing webhook
```

Comment:
The idea is to *not* edit settings.py, but just to create a  `Settings` object where you specify the settings you want to change (e.g. like we do for the integration tests). But this will likely need an update after I've implemented the environment variables, so we can leave it like this for now.